src/wallet/api.py

```diff
@@ -84,30 +99,26 @@ def withdraw(owner_id, amount, reference=None, pending=False):
     if amount > balance:
         raise exceptions.InsufficientFunds
     qs = models.Wallet.objects.get(id=wallet_obj.id).transactions
-    trx_obj = qs.create(
-        trx_type=TrxType.PENDING if pending else TrxType.FINALIZED,
-        amount=-amount,
-        reference=reference
-    )
+    trx_obj = qs.create(amount=(-amount), reference=reference)
```

Comment:
I know, however, they get your attention in noticing that the amount is negative. As the `-` kind of disappears. 
It might be the readers responsibility to notice it, and maybe a comment would be a better option to keep him informed.